docs/development/workflow/additional_git_topics.rst

```diff
@@ -291,7 +291,7 @@ site that does it, at least as of 2/6/2012). There are two options to do this,
 both of which only work if you own the repository or have the ability to commit
 directly to it (for Astropy, that means being an Astropy maintainer):
 
-* You can use the script at https://gist.github.com/1750715, which will
+* You can use the script at https://gist.github.com/eteq/1750715, which will
```

Comment:
I think converting issues to PRs is not a thing we should encourage (and indeed we hardly ever do it), so I suppose this part can be removed altogether. The script in the meantime also ended up in https://github.com/astropy/astropy-procedures, so the relevant part of this comment for this PR is that the link can/should be updated to point to that one instead of the gist.

Reply:
I changed the link but opened a new issue a #9243 to discuss actually removing the whole section.

Reply:
Perfect. Let's merge this then when we have an all green!